pkg/core/session.go

```diff
@@ -207,15 +222,19 @@ func NewControlSession(d drive.DriveIntf, d0 *Level0Discovery, opts ...ControlSe
 	c := NewPlainCommunication(d, hp, tp)
 	s := &ControlSession{
 		Session: Session{
-			d:     d,
-			c:     c,
-			ComID: ComIDInvalid,
-			TSN:   0,
-			HSN:   0,
+			d:               d,
+			c:               c,
+			ComID:           ComIDInvalid,
+			TSN:             0,
+			HSN:             0,
+			ReceiveRetries:  DefaultReceiveRetries,
+			ReceiveInterval: DefaultReceiveInterval,
 		},
 		HostProperties:           hp,
 		TPerProperties:           tp,
 		MaxComPacketSizeOverride: DefaultMaxComPacketSize,
+		ReceiveRetries:           DefaultReceiveRetries,
+		ReceiveInterval:          DefaultReceiveInterval,
```

Comment:
@yogev-lb : why do we need `ReceiveRetries`/`ReceiveInterval` both inside `Session` and inside `ControlSession`? the latter embeds the former. which flow _requires_ the values to be different and why?